native/testnet.env.example

```diff
@@ -4,6 +4,7 @@
 export MEZOD_CHAIN_ID="mezo_31611-1"
 export MEZOD_HOME="/var/mezod"
```

Comment:
It's worth noting that `MEZOD_USER` can have insufficient permissions for `MEZOD_HOME`. This can easily crash the installation, in order to avoid that I suggest adding a line or two about it in the docs.

Reply:
Some way of checking this in the script would also be a requirement. We have to ensure the script runs with proper parameters before even starting the installation.

Reply:
Thank you for the input. The script is still run as root and I added a function set_file_permissions() that runs before the processes are started with systemd.

Did you encounter an issue?